engine/class_modules/priest/sc_priest_shadow.cpp

```diff
@@ -2190,18 +2190,22 @@ void priest_t::generate_apl_shadow()
       "variable,name=all_dots_up,op=set,value="
       "dot.shadow_word_pain.ticking&dot.vampiric_touch.ticking&dot.devouring_plague.ticking" );
   default_list->add_action( "variable,name=searing_nightmare_cutoff,op=set,value=spell_targets.mind_sear>3" );
+  default_list->add_action(
+      "variable,name=pi_vf_condition,op=set,value=level>=58&cooldown.power_infusion.up|level<58&cooldown.void_eruption."
+      "up",
+      "Transition condition so our APL works when you don't have Power Infusion" );
```

Comment:
Can't you give this a text without treating it as transitional, instead trying to explain what it is trying to express? You can of course mention that it is for lower levels without PI available.

Then we can just keep it for a little bit longer and keep lower level sims working